packages/calcite-components/src/components/input-number/input-number.stories.ts

```diff
@@ -216,3 +216,19 @@ export const widthSetToBreakpoints_TestOnly = (): string =>
       value="123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890"
     ></calcite-input-number>
   `);
+
+export const inputNumberPrefixAndSuffixWidth = (): string => html`
```

Comment:
follow https://github.com/Esri/calcite-design-system/wiki/component-tokens#storybook for customTheming instructions for new tokens